llvm/lib/Target/AIE/AIEBaseSubtarget.cpp

```diff
@@ -661,6 +661,8 @@ AIEBaseSubtarget::getPostRAMutationsImpl(const Triple &TT) {
   std::vector<std::unique_ptr<ScheduleDAGMutation>> Mutations;
   Mutations.emplace_back(std::make_unique<LockDelays>());
   if (!TT.isAIE1()) {
+    if (EnableWAWStickyRegisters)
+      Mutations.emplace_back(std::make_unique<WAWStickyRegistersEdges>());
```

Comment:
We need to check the QoR impact of that, this touches everything scheduled by the postmissched, not just the post-pipeliner.

Reply:
I see no worrying changes in QoR.

Reply:
But did you compare against plain `aie-public`? From what I understand the numbers you shared were with the pre-pipeliner disabled. If the latter is enabled, there could be QoR changing when enabling that new mutator for the postmisched as well.